Starting up Uvicon worker has errors and failing App start up

There are some breaking changes introduced to [Uvicorn](https://github.com/Kludex/uvicorn/discussions/2706#discussioncomment-14545648) and that is effecting the Unicorn-worker as well. 

```
2025-09-30 08:44:34 -0400] [40] [ERROR] Exception in worker process
Traceback (most recent call last):
File "/usr/local/lib/python3.13/site-packages/gunicorn/arbiter.py", line 608, in spawn_worker
worker.init_process()
~~~~~~~~~~~~~~~~~~~^^
File "/usr/local/lib/python3.13/site-packages/uvicorn_worker/_workers.py", line 96, in init_process
self.config.setup_event_loop()
~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
File "/usr/local/lib/python3.13/site-packages/uvicorn/config.py", line 479, in setup_event_loop
raise AttributeError(
...<3 lines>...
)
AttributeError: The `setup_event_loop` method was replaced by `get_loop_factory` in uvicorn 0.36.0.
```

The change is only noticed in CCMDev and not locally, since the start up of the app is slightly different in Local and Prod

-------------------
Test Plan:
CCMDev instance should starts up fine and App launches

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,6 +1,6 @@
 # server and workers
 gunicorn==23.0.0
-uvicorn-worker==0.3.0
+uvicorn-worker==0.4.0
 
 mysqlclient==2.2.7
 

